sql/driver/http_post.go

```diff
@@ -73,6 +74,14 @@ func httpPost(c postContext, request, response gogoproto.Message, method fmt.Str
 		return err
 	}
 
+	// TODO(pmattis): Figure out the right thing to do here. The HTTP client we
+	// get back from base.Context has a 3 second timeout. If a client operation
+	// takes longer than that it will be retried. If we're not in an explicit
+	// transaction the auto-transaction created on the server for the second
+	// operation will be different than the still running first operation giving
+	// them the potential to stomp on each other.
+	client.Timeout = 60 * time.Second
```

Comment:
`GetHTTPClient` returns a shared client object, which we shouldn't mutate here. Make a copy first.